src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
 </html>
+
+</html>
```

Comment:
Remove the duplicate closing `</html>` tag. There should only be one closing `</html>` tag at the end of the document.